examples/plot_fiac_analysis.py

```diff
@@ -47,13 +48,15 @@
 data = datasets.fetch_fiac_first_level()
 fmri_files = [data['func1'], data['func2']]
 design_files = [data['design_matrix1'], data['design_matrix2']]
+design_matrices = [pd.DataFrame(np.load(df)['X']) for df in design_files]
+
 
 # Load all the data into a common GLM
 multi_session_model = FirstLevelGLM(data['mask'], standardize=False,
                                     noise_model='ar1')
 
 # GLM fitting
-multi_session_model.fit(fmri_files, design_files)
+multi_session_model.fit(fmri_files, design_matrices)
```

Comment:
Why we have to clarify that is a multi_session_model? it sounds to me like I have to do something different for multi_session and single_session, when in reality is just passing a list or single variables as parameters to glm_model.fit(imgs, design_matrices). Also in some examples we use epi_img, here fmri_files, etc, shouldnt we keep a name convention in the examples, it helps me a lot as a novice user.